<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="VectorPlot" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>551</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="grateDateTime">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="displayFormat">
     <string>yyyy/MM/dd hh:mm AP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Run</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>551</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Bedload">
     <attribute name="title">
      <string>Bedload</string>
     </attribute>
     <widget class="QCustomPlot" name="BedloadPlot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="BankWidth">
     <attribute name="title">
      <string>Bank W</string>
     </attribute>
     <widget class="QCustomPlot" name="BankWidthPlot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>191</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>105</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="plainText">
        <string>Channel Width
Flow Depth (x100)</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>105</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="XSect">
     <attribute name="title">
      <string>Cross Section</string>
     </attribute>
     <widget class="QCustomPlot" name="XSectPlot" native="true">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>421</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bank Ht</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinWidth">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>54</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>57</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Bed Width</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBankHt">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>135</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinNode">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>13</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum">
       <number>130</number>
      </property>
      <property name="value">
       <number>45</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>57</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Node</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinDepth">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>95</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>57</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Depth</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>57</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>D50 (mm)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>D Cmp (mm)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinNoChnl">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>14</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinDcomp">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>95</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinD50">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>55</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>57</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>NoChannels</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>D90</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinD90">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>135</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinTheta">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Theta</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinHmax">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hmax</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="runProgress">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="reportQw">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Discharge m3/s</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="reportStep">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Step</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="QwSeries" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="GSD_Dash" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>260</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>351</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Delta t (time step in seconds)</string>
    </property>
    <widget class="QSlider" name="deltaT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="value">
      <number>12</number>
     </property>
     <property name="sliderPosition">
      <number>12</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dt_disp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="writeInt_disp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Write Interval</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="outputFileName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="overwriteMode">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>RunResults.txt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Results File</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textFileName">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RegimeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Turn on/off automatic adjust of river width"/>
    </property>
    <property name="text">
     <string>Regime Width</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stopRun">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Run</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="reportQs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bedload m3/s</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto-adjust bank</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>width (experimental)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_load_XML"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_load_XML">
   <property name="text">
    <string>Load XML...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
